### DISCUSSION TITLE
Contact Page - New feature - Whatsapp

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#101] [Feedback]
In some websites they have a clickable link that connects to whataspp. Again it is more convenient for phones.

https://faq.whatsapp.com/iphone/how-to-link-to-whatsapp-from-a-different-app/?lang=en

This can be a future feature as we do not have a dedicated whatsapp number yet but are thinking on getting one specifically for communicating with donors.

**Comment by khawarizmus** [comment]
@BennaceurHichem this might be something you want to do
